### PR TITLE
Fix constant OutOfDomainError

### LINKED ISSUE #2: NameError: uninitialized constant RedBlocks::OutOfDomainError Did you mean? FloatDomainError
## WHY
`RedBlocks::OutOfDomainError` is raised at

https://github.com/Altech/red_blocks/blob/c79256c80d2d61c6d1a11b889c84fc3044b16a3e/lib/red_blocks/enum_set.rb#L7

But, it is not defined anywhere...

https://github.com/Altech/red_blocks/search?utf8=%E2%9C%93&q=OutOfDomainError&type=

## WHAT

Maybe it means `DomainError`

https://github.com/Altech/red_blocks/blob/c79256c80d2d61c6d1a11b889c84fc3044b16a3e/lib/red_blocks/domain_error.rb#L2-L11

### PR DESCRIPTION
**Pull request by ngtk**
## WHY
- To fix #2

## WHAT
- Rename `OutOfDomainError` to `DomainError`

### CODE REVIEW

**Review by Altech — APPROVED** [review]
Thank you for renaming!

**Comment by Altech** [comment]
Could you bump the patch version? If so, please change `RedBlocks::VERSION` constant too.

**Comment by ngtk** [comment]
> Could you bump the patch version? If so, please change RedBlocks::VERSION constant too.

Sure.

File: lib/red_blocks/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module RedBlocks
-  VERSION = "0.1.0"
+  VERSION = "0.1.1"
```
Comment (Altech) [inline lib/red_blocks/version.rb:2]: Good job. 👍

**Comment by ngtk** [comment]
Bumped up and checked that all spec has been passed:

```
red_blocks|fix/out-of-domain ⇒ be rake spec
/Users/ngtk/.anyenv/envs/rbenv/versions/2.4.2/bin/ruby -I/Users/ngtk/.anyenv/envs/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/rspec-core-3.6.0/lib:/Users/ngtk/.anyenv/envs/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/rspec-support-3.6.0/lib /Users/ngtk/.anyenv/envs/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/rspec-core-3.6.0/exe/rspec --pattern spec/\*\*\{,/\*/\*\*\}/\*_spec.rb
....................*......................

Pending: (Failures listed here are expected and do not affect your suite's status)

  1) RedBlocks::Set#update! updates only the difference
     # Not yet implemented
     # ./spec/red_blocks/set_spec.rb:70


Finished in 0.04308 seconds (files took 0.32187 seconds to load)
43 examples, 0 failures, 1 pending
```

**Pull request merged**